packages/inline-definition/src/InlineDefinition.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip, { TooltipProps } from '@leafygreen-ui/tooltip';
+import { Body } from '@leafygreen-ui/typography';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const underline = css`
+  display: inline-flex;
+  background-repeat: repeat-x;
+  background-position: center bottom;
+  background-size: 3px 2px;
+  transition: background-image 150ms ease-in-out;
+
+  background-image: radial-gradient(
+    circle closest-side,
+    ${uiColors.gray.dark1} 75%,
+    transparent 25%
+  );
+
+  &:hover {
+    background-image: radial-gradient(
+      circle closest-side,
+      currentColor 75%,
+      transparent 25%
+    );
+  }
+`;
+
+const maxWidth = css`
+  max-width: 240px;
+`;
+
+type InlineDefinitionProps = Partial<TooltipProps> & {
+  definition: string;
+};
+
+function InlineDefinition({
+  definition,
+  children,
+  className,
+  ...tooltipProps
+}: InlineDefinitionProps) {
+  return (
+    <Tooltip
+      justify="middle"
+      spacing={5}
+      {...tooltipProps}
+      trigger={<span className={cx(underline, className)}>{children}</span>}
```

Comment:
I'm wondering if it will be clear what element the className is applied to... Perhaps applying the className to Tooltip, and allowing them to style the element via our minimal styling and their control of children is better? Not sure here.

Reply:
Yeah I think you're probably right, that makes sense.

Reply:
I'm not sure: 

```
<Body>
      Base hourly rate is for a MongoDB{' '}
      <InlineDefinition definition={replicaSetDefintion} className={css`color: pink`}>
        replica set
      </InlineDefinition>{' '}
      with 3 data bearing servers
    </Body>
```

versus 

```
<Body>
      Base hourly rate is for a MongoDB{' '}
      <InlineDefinition definition={replicaSetDefintion}>
       <span className={css`color:pink`}> replica set </span>
      </InlineDefinition>{' '}
      with 3 data bearing servers
    </Body>
```

Reply:
it's an easy change to make but not convinced I like the DX better in the second example. Feels like unnecessary additions to the DOM